source/common/http/conn_manager_utility.cc

```diff
@@ -22,18 +22,30 @@ void ConnectionManagerUtility::mutateRequestHeaders(
     Runtime::RandomGenerator& random, Runtime::Loader& runtime,
     const LocalInfo::LocalInfo& local_info) {
   // Clean proxy headers.
```

Comment:
oops sorry, one more nit: Can you please move this comment down with the rest of header removes?